to-do-app/src/App.js

```diff
@@ -1,23 +1,26 @@
-import logo from './logo.svg';
 import './App.css';
-
+import {BrowserRouter as Router, Switch, Route} from 'react-router-dom'
+import Header from './components/Header'
+import Navbar from'./components/Navbar'
+import AddTask from './components/AddTask';
+import DeleteTask from './components/DeleteTask';
+import DisplayActivities from './components/DisplayActivities';
+import EditTask from './components/EditTask';
+import UpcomingActivities from './components/UpcomingActivities';
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app">
+      <Router>
+      <Header/>
+      <Navbar/>
+        <Switch>
```

Comment:
Somewhere should be a <main> element with the page content except header, navigation and footer.
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main